_data/code/provincial/bc.yml

```diff
@@ -1,37 +1,68 @@
---- 
```

Comment:
@BassamAjam It looks like this is about what I was expecting to see when you submit an edit for an existing application. One thing to keep in mind, is when we write out the YAML file to GitHub, the POJO > string of YAML text process has opinions about how it should be formatted. That is why we see things like schemaVersion having single quotes added. It also seems to have opinions about spacing in these files. Depending on whether these are considered good style for our YAML files to have, we may have to tweak how we write out the YAML file.